x/superfluid/README.md

```diff
@@ -455,7 +455,88 @@ Disable multiple assets from being used for superfluid staking.
 
 ## Events
```

Comment:
I love the motivation for this, but I can see this going out of date and being difficult to maintain.

Reply:
I think we might still want to have it since these docs get auto-uploaded to the docs website. Currently, there is no info about events but it could be useful for integrators. WDYT?